assets/component-facets.css

```diff
@@ -745,7 +745,6 @@ details.menu-opening .mobile-facets__close svg {
   width: 100%;
   bottom: 0;
   left: 0;
-  background-color: rgb(var(--color-background));
   z-index: 3;
   transform: translateX(100%);
```

Comment:
We have two outlines on focus here: grey and the red. Is that expected? 🤔 I dont see this on main

https://screenshot.click/01-09-nc3xl-htdlt.mp4